esp-hal-common/src/clocks_ll/esp32.rs

```diff
@@ -360,16 +360,16 @@ pub(crate) fn set_cpu_freq(cpu_freq_mhz: crate::clock::CpuClock) {
             .store5
             .modify(|_, w| w.scratch5().bits(value as u32));
 
-        esp32_update_cpu_freq(cpu_freq_mhz.frequency().to_Hz());
+        esp32_update_cpu_freq(cpu_freq_mhz.mhz());
     }
 }
 
 /// Set the real CPU ticks per us to the ets, so that ets_delay_us
```

Comment:
This comment needs to be updated if the value is now in mhz instead of ticks per us